src/ast-utils.ts

```diff
@@ -24,7 +24,7 @@ export default class ASTUtils {
     }
   }
 
-  static prependMultiple(target, nodes: ASTNode[]) {
+  private static prependMultiple(target, nodes: ASTNode[]) {
```

Comment:
style: all private names should start with `_` so that it's apparent that it's private-ish after compilation.